setup.py

```diff
@@ -12,9 +12,19 @@
     download_url="https://github.com/trustyou/retwist/archive/v{}.tar.gz".format(version),
     url="https://github.com/trustyou/retwist",
     install_requires=[
-        "twisted",
+        "twisted==18.9.0",
```

Comment:
Can we do this without pinning the twisted version? Think of projects using retwist which are developed against an older twisted version. Do we want to force them to upgrade?

Reply:
Agree with you